analysed-packages/glib-networking/version-2.72.2/glib-networking-2.72.2-OSS-disclosure.txt

```diff
@@ -0,0 +1,2081 @@
+no one other than the license steward has the right to modify or publish new
+versions of this License. Each version will be given a distinguishing version
+number.
+
+10.2. Effect of New Versions
+You may distribute the Covered Software under the terms of the version of the
+License under which You originally received the Covered Software, or under the
+terms of any subsequent version published by the license steward.
+
+10.3. Modified Versions
+If you create software not governed by this License, and you want to create a
+new license for such software, you may create and use a modified version of this
+License if you rename the license and remove any references to the name of the
+license steward (except to note that such modified license differs from this
+License).
+
+10.4. Distributing Source Code Form that is Incompatible With Secondary Licenses
+If You choose to distribute Source Code Form that is Incompatible With Secondary
+Licenses under the terms of this version of the License, the notice described in
+Exhibit B of this License must be attached.
+
+Exhibit A - Source Code Form License Notice
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v.
+2.0. If a copy of the MPL was not distributed with this file, You can obtain one
+at http://mozilla.org/MPL/2.0/.
+
+If it is not possible or desirable to put the notice in a particular file, then
+You may include the notice in a location (such as a LICENSE file in a relevant
+directory) where a recipient would be likely to look for such a notice.
+
+You may add additional accurate notices of copyright ownership.
+
+Exhibit B - "Incompatible With Secondary Licenses" Notice
+
+This Source Code Form is "Incompatible With Secondary Licenses", as defined by
+the Mozilla Public License, v. 2.0.
+
+--------------------------------------------------------------------------------
+
+Copyright notices
+Copyright (C) 1991, 1999 Free Software Foundation, Inc.
+Copyright 2011, 2015, 2016 Collabora, Ltd.
+Copyright (C) 2015 NICE s.r.l.
+Copyright (C) 2021 Ole André Vadla Ravnås <oleavr@frida.re>
+Copyright 2011 Collabora, Ltd.
+Copyright 2009 Red Hat, Inc
+Copyright 2010 Red Hat, Inc
+Copyright 2009 Red Hat, Inc.
+Copyright 2015, 2016 Collabora, Ltd.
+Copyright 2009-2011 Red Hat, Inc
+Copyright 2010 Red Hat, Inc.
+Copyright 2021 Igalia S.L.
+Copyright (C) 2011 Collabora Ltd.
+Copyright 2010 Collabora, Ltd
+Copyright 2018 Igalia S.L.
+Copyright 2021 Red Hat, Inc
+Copyright 2010 Collabora, Ltd.
+Copyright (C) 2011-2012, 2015, 2020, 2022 Free Software Foundation, Inc.
+Takayuki KUSANO <AE5T-KSN@asahi-net.or.jp>, 2011-2012. Hideki Yamane
+<henrich@debian.org>, 2011-2012. Yoji TOYODA <bsyamato@sea.plala.or.jp>, 2012.
+Jiro Matsuzawa <jmatsuzawa@gnome.org>, 2015. sicklylife
+<translation@sicklylife.jp>, 2020, 2022.
+Copyright (C) 2011 Tommi Vainikainen
+Copyright (C) 2011 The GNOME Project.
+Copyright (C) 2011 glib-networking'S COPYRIGHT HOLDER This file is distributed
```

Comment:
Please remove the string 

This file is distributed nder the same licence as the glib-networking package